libs/shinkai-node-state/src/v2/queries/getNodeStorageLocation/useGetNodeStorageLocation.ts

```diff
@@ -0,0 +1,14 @@
+import { useQuery } from "@tanstack/react-query";
+
+import { getNodeStorageLocation } from ".";
+import { GetNodeStorageLocationInput } from "./types";
+
+export const useGetNodeStorageLocation = ({
+  nodeAddress,
+  token,
+}: GetNodeStorageLocationInput) => {
+  return useQuery({
+    queryKey: ["nodeStorageLocation", nodeAddress, token],
```

Comment:
nit: to be consistent, could we add the prefix key in FunctionKeyV2 and pass directly the input (so we have a standard format in the query dev tools)?
```
    queryKey: [FunctionKeyV2.GET_HEALTH, input], 
 ```